answers.txt

```diff
@@ -1,4 +1,14 @@
 •  What is a gitignore file and what should it include for a Java project? 
+It is a text file which is placed in the git repository and conveys the git to not track certain files and folders that one doesn't want to upload to the master repository. .class, .template files that are created by ide at runtime.
+
```

Comment:
Could you please add little bit more about gitignore file? Also you need to specify what should it include for a Java project.

Reply:
Addressed.